main.py

```diff
@@ -1,10 +1,61 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        info_file = json.load(file)
+
+    for player_name, player_info in info_file.items():
+
+        race_dict = player_info["race"]
+        guild_dict = player_info["guild"]
+
+        if not Race.objects.filter(
+            name=race_dict["name"]
+        ).exists():
+            race = Race.objects.create(
+                name=race_dict["name"],
+                description=race_dict["description"]
+            )
+        else:
+            race = Race.objects.get(name=race_dict["name"])
+
+        for skill in race_dict["skills"]:
+            if not Skill.objects.filter(name=skill["name"]).exists():
+                Skill.objects.create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race
+                )
+            else:
+                Skill.objects.get(name=skill["name"])
+
+        if guild_dict is None:
+            guild = None
+        else:
```

Comment:
unnecessary else here, u can implement guild = None, and then check if guild_dict is not None
Also u can use __get_or_create__ method to optimize ur code